### DISCUSSION TITLE
In primeNG 17.13.0 has new issue on table in property expandedRowKeys

### DISCUSSION THREAD

**Discussion by tsoriano64** [primefaces/community#1649] [PrimeNG] [answered]
There is a new bug in the "table" component in the expanded row section if you use the "expandedRowKeys" property.
In version 17.12.0 that did not happen.

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?

**Comment by tsoriano64** [comment]
Sure.

[https://stackblitz.com/edit/d732d8?file=src%2Fapp%2Fdemo%2Ftable-row-expand-demo.html](https://stackblitz.com/edit/d732d8?file=src%2Fapp%2Fdemo%2Ftable-row-expand-demo.html)

**Reply by cetincakiroglu** [reply, marked as answer]
Hi,

This issue is fixed in 17.14.0 https://github.com/primefaces/primeng/issues/15270
Here is your example with 17.14.1 https://stackblitz.com/edit/d732d8-r3jeee

**Comment by tsoriano64** [comment]
Great, thanks for going so fast!!
